sim/src/simulator.rs

```diff
@@ -4,13 +4,16 @@
 use crate::edge::Edge;
 use crate::node::Node;
 use crate::sim_element::SimElement;
+use crate::storage::Storage;
 use petgraph::dot::{Config, Dot};
 use petgraph::graph::{Graph, NodeIndex};
 use std::collections::HashMap;
 use std::fmt::Display;
 use std::fs;
 use std::process::Command;
 
+const STORAGE_ID: &str = "storage";
```

Comment:
I do not think storage should become a fixed part of the simulator. That does not make a whole lot of sense. The simulator should have no notion of a storage. Instead we need to have facilities to connect nodes to storage and allow communication or requests.

Reply:
Yes, it is better for it to be just another sim element like node or edge.  I have made it like that, with its own interface as part of the simulator.